lib/data/local/localizations/language_cache_manager.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:ui';
+import 'package:fluffychat/utils/famedlysdk_store.dart';
+import 'package:matrix/matrix.dart';
+
+class LanguageCacheManager {
+  static const keyLanguageCode = 'LANGUAGE_CODE';
+  static const keyCountryCode = 'COUNTRY_CODE';
+
+  final Store pres = Store();
```

Comment:
Can you clarify what is this, i think you should change name. and should it be singleton ?

Reply:
It belongs to matrix sdk, I reuse it all the time

Reply:
please use getIt.

Reply:
done